src/pages/billing/List/List.ts

```diff
@@ -0,0 +1,257 @@
+            _.map(project => h(ProjectListItem, {
+              project, key: project.projectName, loadProjects,
+              isActive: !!selectedName && project.projectName === selectedName,
+              isCreatingStatus: isCreatingStatus(project.status)
+            }), projectsShared)
+          ])
+        ])
+      ]),
+      creatingBillingProjectType === 'GCP' && h(GCPNewBillingProjectModal, {
+        billingAccounts,
+        loadAccounts,
+        onDismiss: () => setCreatingBillingProjectType(null),
+        onSuccess: billingProjectName => {
+          Ajax().Metrics.captureEvent(Events.billingCreationBillingProjectCreated, {
+            billingProjectName, cloudPlatform: cloudProviderTypes.GCP
+          })
+          setCreatingBillingProjectType(null)
+          loadProjects()
+        }
+      }),
+      div({
+        style: {
+          overflowY: 'auto', flexGrow: 1, display: 'flex', flexDirection: 'column'
+        }
+      }, [Utils.cond(
+        [selectedName && !_.some({ projectName: selectedName }, billingProjects),
+          () => div({
+            style: {
+              margin: '1rem auto 0 auto'
+            }
+          }, [
+            div([
+              h2(['Error loading selected billing project.']),
+              p(['It may not exist, or you may not have access to it.'])
+            ])
+          ])],
+        [azureUserWithNoBillingProjects || creatingAzureBillingProject, () => h(AzureBillingProjectWizard, {
+          onSuccess: billingProjectName => {
+            Ajax().Metrics.captureEvent(Events.billingCreationBillingProjectCreated, {
+              billingProjectName, cloudPlatform: cloudProviderTypes.AZURE
+            })
+            setCreatingBillingProjectType(null)
+            loadProjects()
+          }
+        })],
+        [!isLoadingProjects && _.isEmpty(billingProjects) && !Auth.isAzureUser(), () => h(GCPBillingProjectWizard, {
+          billingAccounts,
+          onSuccess: billingProjectName => {
+            Ajax().Metrics.captureEvent(Events.billingCreationBillingProjectCreated, {
+              billingProjectName, cloudPlatform: cloudProviderTypes.GCP
+            })
+            setCreatingBillingProjectType(null)
+            loadProjects()
+            Nav.history.push({
+              pathname: Nav.getPath('billing', undefined, undefined),
+              search: qs.stringify({ selectedName: billingProjectName, type: 'project' })
+            })
+          },
+          authorizeAndLoadAccounts
+        })],
+        [selectedName && _.some({ projectName: selectedName }, billingProjects), () => {
+          const billingProject = _.find({ projectName: selectedName }, billingProjects)
+          return h(ProjectDetail, {
+            key: selectedName,
+            billingProject,
+            billingAccounts,
+            authorizeAndLoadAccounts,
+            reloadBillingProject: () => reloadBillingProject(billingProject).catch(loadProjects),
```

Comment:
I take this to mean that we are initiating a reload of the billing projects list if the reload of the specific billing project fails. Is that right?

Reply:
Exactly.